### ISSUE TITLE
Simultaneous GM and Player Character Sheet View Causes Lag

### ISSUE THREAD

**Issue by TrofimSelbs** [jessev14/custom-character-sheet-sections#27]
Hey! Thank you for this wonderful module. I have a bit of an issue. When I open the character sheet simultaneously with a player - both they and I start to lag significantly. 
- they can't scroll
- I can't input or change any numbers on their sheet
- some UI elements flicker

I have used find the culprit and found this module. I have made sure to disable every other module and check only this one and it still gave me this result. Could you please look into this? Thank you

Foundry v11 315
DnD 5e 3.1.2

**Comment by jessev14** [comment]
Are there any errors in the console (F12) when this happens?

**Comment by TrofimSelbs** [comment]
Hey, I hopped on a call with my player and verified that this is still an issue. Here are screenshots and files from our chrome console. 

These are from the player:
![image](https://github.com/jessev14/custom-character-sheet-sections/assets/124184162/5d7ce465-809f-4cd5-aa47-bf7c7b2f023c)
[ice-wind-dolina.eu.forge-vtt.com-1716118827529.log](https://github.com/jessev14/custom-character-sheet-sections/files/15369187/ice-wind-dolina.eu.forge-vtt.com-1716118827529.log)


These are mine (DM:
![SCR-20240519-mewm](https://github.com/jessev14/custom-character-sheet-sections/assets/124184162/a0dd8c3f-1354-44fe-b2a9-e9e6f9c689d6)
[ice-wind-dolina.eu.forge-vtt.com-1716118817109.log](https://github.com/jessev14/custom-character-sheet-sections/files/15369182/ice-wind-dolina.eu.forge-vtt.com-1716118817109.log)


These readings were taken with every module except following turned off with find culprit:
![image](https://github.com/jessev14/custom-character-sheet-sections/assets/124184162/96d6dacd-380b-44c0-83f6-b57e344c4bd3)

**Comment by jessev14** [comment]
Thanks for the info. What browsers were being used?

**Comment by TrofimSelbs** [comment]
My player and I used foundry through Forge in Chrome. We used Chrome “applications” (it is a Chrome feature that makes a website into a separate window and calls it an app). Two other players experienced the same issue in plain Chrome and in Opera.

**Comment by jessev14** [comment]
Would you be able to test this without the Forge?

**Comment by TrofimSelbs** [comment]
Unfortunately no, :( I don’t know how to self host to see if it raises this issue

**Comment by ChobitsCrazy** [comment]
I can confirm that this happens without Forge, using Edge browser.
Only modules enabled are CCSS and libWrapper
Player and myself as the GM have the same character sheet open and causes flickering.
![2024-06-10_22-28-01](https://github.com/jessev14/custom-character-sheet-sections/assets/21301107/10ff6d53-1d20-481c-b0d8-27ab698c7e70)

**Comment by TrofimSelbs** [comment]
@ChobitsCrazy I have the exact same flicker. Thanks for commenting on the issue

**Comment by Zanderaf** [comment]
Yea, have the same issue self-hosted and with chrome

**Comment by ChobitsCrazy** [comment]
Okay after some grueling testing I have recreated the issue without fail. @enso 
FVTT: v11.315
D&D: 3.1.2
CCSS: 2.1.0
I can recreate completely on my own machine. I have the foundry desktop application opened and minimized. I use Edge for my GM session logged into a world only active modules are CCSS and libWrapper. I use Brave browser as the client who is logged into as a simple player.
Create a brand new PC actor.
Create a new item and give it a custom section.
Give the item to the new actor.
Delete the item from the actor.
Open the character sheet as the GM and as a player in seperate sessions.
Navigate to the Bio tab while on edit and you will see that the character sheet is constantly refreshing.
The source of the issue I found is related to the flags in the actor json. It adds these null values:
```js
"flags": {
    "custom-character-sheet-sections": {
      "sectionOrder-inventory": [
        "all",
        null,
        null
      ],
      "sectionOrder-features": [
        "active",
        "passive",
        "other"
      ]
    },
```
[fvtt-Actor-test-XPM5sAajkAIGTp8T.json](https://github.com/user-attachments/files/15960684/fvtt-Actor-test-XPM5sAajkAIGTp8T.json)
